easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,34 @@
+# Author:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'muParser'
+version = '2.2.5'
+
+homepage = 'http://beltoforion.de/article.php?a=muparser'
+
+description = """
+ muParser is an extensible high performance math expression parser library
+ written in C++. It works by transforming a mathematical expression into
+ bytecode and precalculating constant parts of the expression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://github.com/beltoforion/muparser/archive/v2.2.5.tar.gz
+source_urls = ['https://github.com/beltoforion/muparser/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+sanity_check_paths = {
+    'files': ['include/muParser.h', 'lib/libmuparser.so'],
+    'dirs': [],
+}
+
+parallel = 1
```

Comment:
Why serial build? The `foss-2015a` version of the same build seem to be fine with parallel build...